isis/src/hayabusa/apps/amica2isis/AmicaImportFits.cpp

```diff
@@ -19,7 +19,7 @@
  *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
  *   http://www.usgs.gov/privacy.html.
  */
-#include "ImportFits.h"
+#include "AmicaImportFits.h"
```

Comment:
ImportFits was mostly duplicated between Amica and Nirs but there is a logicblock that is different. Need to consolidate the two and move into the ISIS3 lib.